### ISSUE TITLE
Support for Whisper JAX

### ISSUE THREAD

**Issue by nicholaskoerfer** [ahmetoner/whisper-asr-webservice#175]
Hi, thanks for this Software. It works perfect with Bazarr, for subtitle Translation.

However, it is, even with Faster-Whisper and a Tesla T4 GPU a bit slow.
There is a new Whisper implementation, that can Transcribe 1 Hour of Audio in approx. 15 seconds.

The new Whisper implementation is called "Whisper JAX" (https://github.com/sanchit-gandhi/whisper-jax). It has support for CPU, GPU and even TPU, though there is already a big speed gain just by using a GPU.

Is it possible for you to add this whisper implementation to this ASR Webservice, so one can select between OpenAI Whisper, Faster Whisper and Whisper JAX?

Thanks!

**Comment by Jessomadic** [comment]
This seems very interesting if it's easy to implement

**Comment by Jessomadic** [comment]
@nicholaskoerfer How is the accuracy of it? It's not worth gaining speed if the subtitles are worse

**Comment by Jessomadic** [comment]
HuggingFace has a webversion up and holy crap it's fast. 21 min youtube video in about 8 seconds

**Comment by nicholaskoerfer** [comment]
Yes, it's incredibly fast. In my opinion, the subtitles are accurate.

**Comment by Jessomadic** [comment]
Is this going to be worked on? @ahmetoner

**Comment by blundercode** [comment]
Hmm, this is pretty interesting great find! Would be cool to have a near-instant service even if the quality is slightly worse.

**Comment by johnnyfleet** [comment]
:+1: from me
